scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RewriteTrailingCommas.scala

```diff
@@ -36,7 +36,6 @@ case object RewriteTrailingCommas extends Rewrite {
       case t: Ctor.Primary => afterEachArgs(t.paramss)
 
       case t: Term.Apply => afterArgs(t.args)
-      case t: Type.Apply => afterArgs(t.args)
```

Comment:
the other copy shows up above, line 31. github doesn't allow commenting on that line :)